generic/tests/jumbo.py

```diff
@@ -49,6 +50,14 @@ def run(test, params, env):
     if guest_ip is None:
         raise error.TestError("Could not get the guest ip address")
 
+    host_ifaces = utils_net.get_host_iface()
+    host_mtu_cmd = "ifconfig %s mtu %s"
+    logging.info("Change all host NICs MTU to 9000")
```

Comment:
I think it's better to use "logging.info("Change all host NICs MTU to %s" % mtu)"